README.md

```diff
@@ -167,8 +167,28 @@ public class InvoiceRow : Something
 [Database]
 public class InvoiceInvoiceRow : Relation
 {
-    public Invoice Invoice;
-    public InvoiceRow InvoiceRow;
+    public Invoice Invoice
+    {
+        get
+        {
+            return this.From as Invoice;
+        }
+        set
+        {
+            this.From = value;
+        }
+    }
+    public InvoiceRow InvoiceRow
```

Comment:
Please add a space line between the methods declaration and the inline comments.

Reply:
Done.

Reply:
Please add a space line between the methods declaration **and the inline comments**.